tools/shareable-examples/src/utils/get-example-module.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { readdirSync } from 'fs';
+import { basename, join, resolve } from 'path';
+import { tsCreateSourceFile } from 'tools/shared/src/utils';
```

Comment:
please use correct package path to import from `'@dynatrace/barista-components/tools/shared'`